suite/tests/CMakeLists.txt

```diff
@@ -4245,7 +4245,7 @@ if (BUILD_CLIENTS)
         set (${testname_full}_postcmd
           "${cmd_pfx}chmod@777@-R@${testname_full}.*.dir")
         set(${testname_full}_postcmd2
-          "firstglob@${cmd_pfx}${drpt2trace_path}@-mode@ELF@-raw_pt@${testname_full}.*.dir/kernel.raw/*.1.pt@-raw_pt_metadata@${testname_full}.*.dir/kernel.raw/*.1.pt.metadata@-elf@${testname_full}.*.dir/kernel.raw/kcore")
+          "firstglob@${cmd_pfx}${drpt2trace_path}@-mode@ELF@-raw_pt_format@DRMEMTRACE@-raw_pt@${testname_full}.*.dir/kernel.raw/*.pt@-elf@${testname_full}.*.dir/kernel.raw/kcore")
```

Comment:
So this test collects a PT trace using drmemtrace, and then decodes using pt2trace. So when we have the raw2trace support this should decode using raw2trace as well right? Add a TODO.

Reply:
Done.